src/components/templates/FireBarrel/FireBarrel.tsx

```diff
@@ -89,7 +88,8 @@ const StyledChair = styled.div<{
     bottom: 1em;
     left: 0;
 
-    text-align: center;
+    text-align: center;import { VideoParticipant } d2dfrom "components/organisms/Video";
```

Comment:
uhm, seems wrong

Reply:
@colinhowe I thought we're removing Styled components.

@alexey-petrov this is definitely wrong. Looks like a missing quote somewhere made the IDE concatenate code into the string.

Reply:
Accidental commit. Yes we are removing styled components - in the redesign. This set of changes is deliberately as minimal as can be.

Resolved in 77323859b